virttest/utils_libvirtd.py

```diff
@@ -29,24 +30,51 @@ class Libvirtd(object):
     Class to manage libvirtd service on host or guest.
     """
 
-    def __init__(self, session=None):
+    def __init__(self, service_name=None, session=None):
         """
         Initialize an service object for libvirtd.
 
+        :params service_name: Service name such as virtqemud or libvirtd.
```

Comment:
It is better to add other sub-daemons, too as this is a new enhancement, so we'd better provide more details. We can also add similar sentences like:
If service_name is None, all sub daemons will be operated when modular daemon environment is enabled,
If service_name is single string, only this daemon/service specified will be operated.

Reply:
Nice, updated.